tests/migration/migration.go

```diff
@@ -3142,15 +3141,8 @@ var _ = SIGMigrationDescribe("VM Live Migration", func() {
 			// So let's be sig-compute and skip ourselves on sig-compute always... (they have only 1 node with CPU manager)
 			checks.SkipTestIfNotEnoughNodesWithCPUManager(2)
 			virtClient = kubevirt.Client()
-
-			By("getting the list of worker nodes that have cpumanager enabled")
-			nodeList, err := virtClient.CoreV1().Nodes().List(context.TODO(), metav1.ListOptions{
-				LabelSelector: fmt.Sprintf("%s=,%s=%s", workerLabel, "cpumanager", "true"),
-			})
-			Expect(err).ToNot(HaveOccurred())
-			Expect(nodeList).ToNot(BeNil())
-			nodes = nodeList.Items
-			Expect(len(nodes)).To(BeNumerically(">=", 2), "at least two worker nodes with cpumanager are required for migration")
+			nodes = checks.GetWorkerNodesWithCPUManagerEnabled(virtClient)
+			checks.ExpectAtLeastTwoNodes(nodes, checks.DescriptionTwoWorkerNodesWCPUManagerRequired)
```

Comment:
Isn't this just `ExpectAtLeastTwoWorkerNodesWithCPUManager`? Also above we use `checks.SkipTestIfNotEnoughNodesWithCPUManager(2)`